spec/features/posts_spec.rb

```diff
@@ -0,0 +1,88 @@
+require 'rails_helper'
+RSpec.describe 'Posts' do
+  describe 'Posts #index' do
+    before do
+      visit user_posts_path(User.first)
+      @posts = User.first.posts
+    end
+    it 'Should be able to view profile picture' do
+      expect(page).to have_css("img[src*='#{User.first.photo}']")
+    end
+
+    it 'Should be able to view the user\'s name' do
+      expect(page).to have_content(User.first.name)
+    end
+
+    it 'Should be able to see the num of posst the user has written' do
+      expect(page).to have_content("Number of posts: #{User.first.post_counter}")
+    end
+
+    it 'Should be able to see the post\'s title.' do
+      @posts.each do |post|
+        expect(page).to have_content(post.title)
+      end
+    end
+
+    it 'Should be able to some of the post\'s body.' do
+      @posts.each do |post|
+        expect(page).to have_content(post.text)
+      end
+    end
+
+    it 'Should be able to see the first comments on a post.' do
+      @posts[0].last_5_comments.each do |comment|
+        expect(page).to have_content(comment.text)
+      end
+    end
+
+    it 'Should be able to see how many likes a post has.' do
+      @posts.each do |post|
+        expect(page).to have_content("Comments: #{post.comment_counter}")
+      end
+    end
+
+    it 'Should be able to see a section for pagination if there are more posts than fit on the view.' do
+      expect(page).to have_button('Pagination')
+    end
+
+    it 'Should be able to click on a post, it redirects me to that post\'s show page.' do
+      click_link(@posts[0].title)
+      expect(page).to have_current_path(user_post_path(@posts[0].author, @posts[0]))
+    end
```

Comment:
- [ ]  As required for the project, kindly ensure that the Post index page has the following featured test(s):
  * Tests that asserts you can see how many comments a post has.

Reply:
@tresorsawasawa  thanks for noticing, we are adding the missing test